### ISSUE TITLE
div is invisible

### ISSUE THREAD

**Issue by AAverin** [junkboy0315/react-compare-image#9]
Hey, thanks for the script

I am using Gatsby.js 1.9, and on release div with comparison gets "display: none" style for some reason. On debug everything works fine
Any ideas why does this happen?

**Comment by junkboy0315** [comment]
I doubt that this line maybe involves.
https://github.com/junkboy0315/react-compare-image/blob/master/src/ReactCompareImage.js#L285

This script waits rendering until the images are fully loaded.

It's better if you can provide some sample code.
I'm also using gatsby and it sometimes behaves odd though...

**Comment by AAverin** [comment]
Surprisingly enough, it seems to work on when I open the site from my Android phone, but not on desktop Chrome
There is not much to provide as a sample. I use Bootstrap and have put before-after inside a column:
```
<div className="row overview" align="center">
                        <div className="col-sm-12 col-md-6">
                            <ReactCompareImage leftImage="/res/OrtonBefore.jpg" rightImage="/res/OrtonAfter_Heavy.jpg" />
                        </div>
                        <div className="col-sm-12 col-md-6">
                            <ReactCompareImage leftImage="/res/OrtonBefore.jpg" rightImage="/res/OrtonAfter_Soft.jpg" />
                        </div>
                    </div>
```

**Comment by junkboy0315** [comment]
@AAverin 

Hi, I changed some logic of this component and released as v0.7.1.

Could you try this one and I hope this solves your problem.

If it doesn't fix your problem, tell me that.
I have one more idea to solve the problem.

**Comment by AAverin** [comment]
@junkboy0315 I have updated to 0.8.1 and looks like it's same so far.
I am also on latest Gatsby and React now.
When I build my static website, before-after doesn't show up. Seems to work when I am in develop mode though.
